types/customize-cra/tsconfig.json

```diff
@@ -22,7 +22,8 @@
         },
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
needed to resolve external dependency modules. see [previous error logs](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/d4a36c4af0b08d724b34c58f7b88a26c74057855/checks?check_suite_id=10019408996#step:6:358)

Reply:
this is becuase of ESM import and 'esModuleInterop' usage in the lib, 'impot * as' woould not impact webpack imports in `.d.ts`. I understand that is not feasable to make this specific import backward compatible (import * as)?